static/src/javascripts/projects/common/modules/commercial/sticky-top-banner.js

```diff
@@ -76,8 +76,11 @@ define([
         var $iframe = getAdIframe();
         var slotWidth = $iframe.attr('width');
         var slotHeight = $iframe.attr('height');
+        var slotSize = [slotWidth, slotHeight].join(',');
```

Comment:
I think the idea is that most of the adverts in question apply height properties to the `$adSlot`. It might be nicer if the creative object itself could be responsible for this and exposing events.